Library/include/CSP/Systems/Assets/AssetSystem.h

```diff
@@ -76,6 +76,18 @@ class CSP_API CSP_NO_DISPOSE AssetSystem : public SystemBase
 	/// @param Callback NullResultCallback : callback when asynchronous task finishes
 	CSP_ASYNC_RESULT void DeleteAssetCollection(const AssetCollection& AssetCollection, NullResultCallback Callback);
 
+	/// @brief Copies an array of asset collections to another space. Note that all source asset collections must belong to the same space.
+	/// @param SourceAssetCollections csp::common::Array<AssetCollection> : The array of asset collections to copy. They must all belong to the same
+	/// space.
+	/// @param DestSpaceId  const csp::common::String : The unique identifier of the space to copy these asset collections to.
+	/// @param CopyAsync const csp::common::Optional<bool> : Whether to instruct the services to perform the copy of the asset collections
+	/// asynchronously.
+	/// @param Callback NullResultCallback : callback when asynchronous task finishes
+	CSP_ASYNC_RESULT void CopyAssetCollectionsToSpace(csp::common::Array<AssetCollection>& SourceAssetCollections,
```

Comment:
What is this in aid of? I see this PR has no ticket, so is this for a personal project?

Reply:
It's related to the upcoming asset library feature, which I have fleshing out in Unreal to help guide the PRD.

Reply:
I would prefer if we had a PRD ready to review before making changes required for said PRD.

Reply:
Hey! The PRD is already in review, and is pretty clear on the requirement for asset duplication, so this is definitely required.